docs/contents/user-docs/irsa.md

```diff
@@ -77,33 +50,61 @@ read -r -d '' TRUST_RELATIONSHIP <<EOF
 EOF
 echo "${TRUST_RELATIONSHIP}" > trust.json
 
-# update variables as needed
+# Update the service name variables as needed
 AWS_SERVICE_NAME='elasticache'
 ACK_CONTROLLER_IAM_ROLE="ack-${AWS_SERVICE_NAME}-controller"
 ACK_CONTROLLER_IAM_ROLE_DESCRIPTION='IRSA role for ACK $AWS_SERVICE_NAME controller deployment on EKS cluster using Helm charts'
 aws iam create-role --role-name "${ACK_CONTROLLER_IAM_ROLE}" --assume-role-policy-document file://trust.json --description "${ACK_CONTROLLER_IAM_ROLE_DESCRIPTION}"
 ```
+Take note of the Amazon Resource Name (ARN) that is associated with your new ACK service controller IAM role. You will need this ARN for later steps. 
 
-#### 2(b) - Attach IAM policy to role
+### Attach an IAM policy to the IAM role
 ```
-# This example uses pre-existing policy for ElastiCache
+# This example uses a pre-existing policy for ElastiCache
 # Create an IAM policy and use its ARN and update IAM_POLICY_ARN variable as needed
 IAM_POLICY_ARN='arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess'
 aws iam attach-role-policy \
     --role-name "${ACK_CONTROLLER_IAM_ROLE}" \
     --policy-arn "$IAM_POLICY_ARN"
 ```
 
-### 3. Associate an IAM role to service account
+For detailed instructions, refer to Amazon EKS documentation on [creating an IAM role and policy for your service account][4].
 
-For detailed instructions, follow instructions at [Associate an IAM role to a service account ][5].
+## Step 3. Associate an IAM role to a service account
 
-#### 3(a) - If Helm charts available on local file system
+If you [installed your ACK service controller using a Helm chart][7], then a service account already exists on your cluster. However, it is still neccessary to associate an IAM role with the service account. 
 
-Update `values.yaml` and set value for `aws.region`, `serviceAccount.annotations`.
+Verify that your service account exists using `kubectl describe`:
+```
+kubectl describe serviceaccount/$ACK_K8S_SERVICE_ACCOUNT_NAME -n $ACK_K8S_NAMESPACE
+```
+Note that the Amazon Resource Name (ARN) of the IAM role that you created is not yet set as an annotation for the service account. 
+
+Use the following commands to associate an IAM role to a service account:
+```
+# Annotate the service account with the ARN
+ISRA_ROLE_ARN=eks.amazonaws.com/role-arn=<IAM Role Arn>
+kubectl annotate serviceaccount -n $ACK_K8S_NAMESPACE $ACK_K8S_SERVICE_ACCOUNT_NAME $ISRA_ROLE_ARN
+```
 
+If you have not done so already, update the AWS region of your ACK service controller using the following commands:
 ```
-# update variables as needed
+# Update your controller with the correct AWS region. Example: us-east-1
+AWS_SERVICE_NAME='elasticache'
+ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
+AWS_REGION=<aws region id>
```

Comment:
all need `export`. Also should we just move `AWS_REGION` up to the first code block, and then reference it instead of using angle brackets in the rest of the sections?